contrib/tcsh/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "tcsh"
+pkgver = "6.24.13"
+pkgrel = 0
+build_style = "configure"
```

Comment:
this looks like gnu_configure and not configure

Reply:
Makes sense! Changing it.